monitoring/grafana/dashboard_msg_validation.json

```diff
@@ -737,46 +822,13 @@
             ]
           }
         },
-        "overrides": [
-          {
-            "__systemRef": "hideSeriesFrom",
-            "matcher": {
-              "id": "byNames",
-              "options": {
-                "mode": "exclude",
-                "names": [
-                  "duplicated proposal with different data",
-                  "late message",
-                  "message round is too far from estimated",
-                  "no duty for this epoch",
-                  "round is too high for this role",
-                  "signer has already advanced to a later slot",
-                  "too many messages of same type per round",
-                  "unknown validator",
-                  "validator is not attesting"
-                ],
-                "prefix": "All except:",
-                "readOnly": true
-              }
-            },
-            "properties": [
-              {
-                "id": "custom.hideFrom",
-                "value": {
-                  "legend": false,
-                  "tooltip": false,
-                  "viz": true
-                }
-              }
-            ]
-          }
-        ]
```

Comment:
@zktaiga there seem to be a few changes like this that aren't related to what's being done in this PR, are they not consequential?

Reply:
> @zktaiga there seem to be a few changes like this that aren't related to what's being done in this PR, are they not consequential?

Maybe it should have been mentioned but I've just verified and these are just minor fixes which are just generic QoL improvements. In this particular case, a list was defaulting a specific item menu, and now it just shows all the items in the list.

Overall the change looks good to me.